packages/editor/src/client/panel/mapLeafletEditPanel.tsx

```diff
@@ -0,0 +1,213 @@
+import React, {useEffect, useState} from 'react'
+import {MaterialIconClose} from '@karma.run/icons'
+import {MarkerPoint} from '../utility'
+import axios from 'axios'
+
+import {
+  NavigationButton,
+  Panel,
+  PanelHeader,
+  PanelSection,
+  TextInput,
+  Spacing,
+  ListValue,
+  ListInput,
+  Box,
+  FieldProps,
+  AutocompleteInput,
+  AutocompleteInputListProps,
+  SelectList,
+  SelectListItem,
+  MarginProps
+} from '@karma.run/ui'
+
+import {MapLeafletItem} from '../blocks/types'
+import nanoid from 'nanoid'
+
+export interface MapLeafletEditPanel {
+  initialItems: ListValue<MapLeafletItem>[]
+  onClose?(items: ListValue<MapLeafletItem>[]): void
+}
+
+export function MapLeafletEditPanel({onClose, initialItems}: MapLeafletEditPanel) {
+  const defaultItemValue = {
+    address: 'Brauerstrasse 42, Zürich',
+    lat: 47.3778762,
+    lng: 8.5271078,
+    title: '',
+    description: '',
+    image: null
+  }
+  const [items, setItems] = useState<ListValue<MapLeafletItem>[]>(() =>
+    initialItems.map(item =>
+      item.value.address === ''
```

Comment:
why this check?

Reply:
this logic basically checks if we're creating a new markerpoint (then use the default item values) or if we're dealing with a marker-point that has already been set. This answer is also tied to my previous one (avoiding the "empty-chip-problem).